### PR TITLE
Fix: DOCtor-RST build

### PR DESCRIPTION
**Pull request by OskarStark**

### CODE REVIEW

**Comment by fabpot** [comment]
Thank you @OskarStark.

File: doc/intro.rst [hunk 1]
```diff
@@ -35,7 +35,7 @@ The recommended way to install Twig is via Composer:
 
 .. code-block:: bash
 
-    composer require "twig/twig:^2.0"
+    composer requireuire "twig/twig:^2.0"
```
Comment (apfelbox) [inline doc/intro.rst:38]: https://github.com/twigphp/Twig/pull/3435

File: doc/filters/spaceless.rst [hunk 2]
```diff
@@ -1,10 +1,6 @@
 ``spaceless``
 =============
 
-.. versionadded:: 1.38
```
Comment (stof) [inline doc/filters/spaceless.rst:4]: this should actually say `2.7.0`

File: doc/intro.rst (hunk 1 above)
Comment (stof) [inline doc/intro.rst:38]: typo

File: doc/filters/spaceless.rst (hunk 2 above)
Comment (OskarStark) [reply doc/filters/spaceless.rst:4]: I created a PR: https://github.com/twigphp/Twig/pull/3436

**Pull request merged**